vlrouter.go

```diff
@@ -99,39 +101,39 @@ func NewVlrouter(agent *OfnetAgent, rpcServ *rpc.Server) *Vlrouter {
 	return vlrouter
 }
 
-// Handle new master added event
-func (self *Vlrouter) MasterAdded(master *OfnetNode) error {
+// MasterAdded handles new master added event
+func (vl *Vlrouter) MasterAdded(master *OfnetNode) error {
 
 	return nil
 }
 
 // Handle switch connected notification
```

Comment:
I guess you are looking at MasterAdded function. I'm referring to the function below(SwitchConnected) and others :)